src/Toast.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\bootstrap4;
+
+use Yii;
+use yii\helpers\ArrayHelper;
+
+/**
+ * Toasts renders an toast bootstrap component.
+ *
+ * For example,
+ *
+ * ```php
+ * echo Toast::widget([
+ *     'title' => 'Hello world!',
+ *     'dateTime' => 'now',
+ *     'body' => 'Say hello...',
+ * ]);
+ * ```
+ *
+ * The following example will show the content enclosed between the [[begin()]]
+ * and [[end()]] calls within the toast box:
+ *
+ * ```php
+ * Toast::begin([
+ *     'title' => 'Hello world!',
+ *     'dateTime' => 'now'
+ * ]);
+ *
+ * echo 'Say hello...';
+ *
+ * Toast::end();
+ * ```
+ *
+ * @see https://getbootstrap.com/docs/4.5/components/toasts/
+ * @author Simon Karlen <simi.albi@outlook.com>
+ */
+class Toast extends Widget
+{
+    /**
+     * @var string the body content in the alert component. Note that anything between
+     * the [[begin()]] and [[end()]] calls of the Toast widget will also be treated
+     * as the body content, and will be rendered before this.
+     */
+    public $body;
+    /**
+     * @var string The tile content in the toast.
+     */
+    public $title;
+    /**
+     * @var int|string|\DateTime|\DateTimeInterface|\DateInterval|false The date time the toast message references to.
```

Comment:
```suggestion
     * @var int|string|\DateTimeInterface|\DateInterval|false The date time the toast message references to.
```

Reply:
According https://github.com/yiisoft/yii2/blob/master/framework/i18n/Formatter.php#L911DateTime is allowed to

Reply:
Yes but `DateTime` is `DateTimeInterface`